timer.js

```diff
@@ -27,11 +36,32 @@ function printEndMessage () {
 function setTimer () {
   return setInterval(function () {
     currentMinute = Math.floor(currentSecond / 60);
-    
-    printTime();
+    if (paused) {
+      printPaused();
```

Comment:
Since the data that we're printing is static, can we only call this function once, and not each time the callback runs?